src/main/webapp/script.js

```diff
@@ -0,0 +1,40 @@
+function loadBody() {
+  verifyLogin();
+}
+
+/** Contacts Login servlet to verify if user is logged in. */
+function verifyLogin() {
+  fetch('/login').then(response => response.json()).then((userDetails => {
+    const loginElement = document.getElementById("login-box");
+    const schoolName = document.getElementById("school-name");
+    if (!userDetails.loggedIn) {
+      schoolName.style.display = "none";
+      loginElement.innerHTML += "<p><a href=\"" + userDetails.url + "\"> Click here to log in! </a></p>";
```

Comment:
Best practice is to use string literals here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals